src/network/simulated/core.rs

```diff
@@ -80,7 +81,10 @@ impl SimulatedNetworkCore {
                     let msg = guard.pop().unwrap();
                     let n_guard = n.read().await;
                     let channel = n_guard.get(&msg.to).unwrap();
-                    channel.send(msg).await.unwrap();
+                    if let Err(_e) = channel.send(msg).await {
+                        println!("sending failed. Ignoring");
```

Comment:
should we only enable this in tests then? to avoid noisy output
```suggestion
                        #[cfg(test)]
                        println!("sending failed. Ignoring");
```

Reply:
makes sense.